app/src/main/res/layout/fragment_main_page.xml

```diff
@@ -16,67 +17,269 @@
+                                android:paddingLeft="@dimen/dp_2"
+                                android:paddingRight="@dimen/dp_5"
+                                android:paddingStart="@dimen/dp_2"
+                                >
+                                <Button
+                                    android:layout_width="match_parent"
+                                    android:layout_height="wrap_content"
+                                    android:background="@drawable/button_yellow_light"
+                                    android:text="@string/com"
+                                    android:textColor="@color/color_white"
+                                    />
+                                <Button
+                                    android:layout_width="match_parent"
+                                    android:layout_height="wrap_content"
+                                    android:layout_marginTop="@dimen/dp_5"
+                                    android:background="@drawable/button_red"
+                                    android:text="@string/nep_cam"
+                                    android:textColor="@color/color_white"
+                                    />
+                            </LinearLayout>
+                        </LinearLayout>
+                    </LinearLayout>
+                    <RelativeLayout
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_marginTop="@dimen/dp_10"
+                        android:background="@drawable/border_layout"
+                        >
+                        <TextView
+                            android:layout_width="@dimen/dp_250"
+                            android:layout_height="wrap_content"
+                            android:layout_marginEnd="@dimen/dp_30"
+                            android:layout_marginRight="@dimen/dp_30"
+                            android:drawableLeft="@drawable/ic_local_dining_24dp"
+                            android:drawablePadding="@dimen/dp_5"
+                            android:drawableStart="@drawable/ic_local_dining_24dp"
+                            android:fontFamily="@string/font_serif"
+                            android:gravity="center_vertical"
+                            android:padding="@dimen/dp_5"
+                            android:text="@string/top_product"
+                            android:textColor="@color/color_orange_mos"
+                            android:textSize="@dimen/sp_17"
+                            android:textStyle="bold"
+                            tools:ignore="UnusedAttribute"
+                            />
+                        <ImageButton
+                            android:layout_width="@dimen/dp_35"
+                            android:layout_height="@dimen/dp_35"
+                            android:layout_alignParentEnd="true"
```

Comment:
ERROR: [RtlCompat] To support older versions than API 17 (project specifies 14) you should *also* add `android:layout_alignParentRight="true"`
 API 17 adds a `textAlignment` attribute to specify text alignment. However, if you are supporting older versions than API 17, you must *also* specify a gravity or layout_gravity attribute, since older platforms will ignore the `textAlignment` attribute.